pages/stylesheets/all.scss

```diff
@@ -20,3 +20,8 @@ margin-bottom: 0px;
   font-size: 1em !important;
   line-height: 1.2em !important;
 }
+.inlineLink {
+  color: #00513f;; 
+  cursor: pointer;
+  text-decoration: underline;
+}
```

Comment:
Can we use the `govuk-link` class for this rather than a custom class? I see the element is a `span` rather than an `a` - but it might still work?

Reply:
govuk-link is now being used